src/OctoshiftCLI.Tests/ado2gh/Commands/MigrateRepo/MigrateRepoCommandHandlerTests.cs

```diff
@@ -188,6 +188,37 @@ public async Task Happy_Path_With_Wait()
         _mockGithubApi.Verify(x => x.GetMigration(MIGRATION_ID));
     }
 
+    [Fact]
+    public async Task Throws_Decorated_Error_When_Create_Migration_Source_Fails_With_Permissions_Error()
+    {
+        // Arrange
+        _mockEnvironmentVariableProvider
+           .Setup(m => m.TargetGithubPersonalAccessToken(It.IsAny<bool>()))
+           .Returns(GITHUB_TOKEN);
+        _mockEnvironmentVariableProvider
+            .Setup(m => m.AdoPersonalAccessToken(It.IsAny<bool>()))
+            .Returns(ADO_TOKEN);
+
+        _mockGithubApi.Setup(x => x.GetOrganizationId(GITHUB_ORG).Result).Returns(GITHUB_ORG_ID);
+        _mockGithubApi
+            .Setup(x => x.CreateAdoMigrationSource(GITHUB_ORG_ID, null).Result)
+            .Throws(new OctoshiftCliException("monalisa does not have the correct permissions to execute `CreateMigrationSource`"));
+
+        // Act
+        await _handler.Invoking(async x => await x.Handle(new MigrateRepoCommandArgs
+        {
+            AdoOrg = ADO_ORG,
+            AdoTeamProject = ADO_TEAM_PROJECT,
+            AdoRepo = ADO_REPO,
+            GithubOrg = GITHUB_ORG,
+            GithubRepo = GITHUB_REPO,
+            Wait = true,
+        }))
+            .Should()
+            .ThrowAsync<OctoshiftCliException>()
+            .WithMessage($"monalisa does not have the correct permissions to execute `CreateMigrationSource`. Please check that (a) you are a member of the `{GITHUB_ORG}` organization, (b) you are an organization owner or you have been granted the migrator role and (c) your personal access token has the correct scopes. For more information, see https://docs.github.com/en/migrations/using-github-enterprise-importer/preparing-to-migrate-with-github-enterprise-importer/managing-access-for-github-enterprise-importer.");
```

Comment:
This console output is nearly 500 columns wide :sweat_smile: Could we add some newlines in the console output so that it looks something like this?

```
monalisa does not have the correct permissions to execute `CreateMigrationSource`
Please check that:
  (a) you are a member of the `{GITHUB_ORG}` organization.
  (b) you are an organization owner or you have been granted the migrator role.
  (c) your personal access token has the correct scopes.
For more information, see https://docs.github.com/en/migrations/using-github-enterprise-importer/preparing-to-migrate-with-github-enterprise-importer/managing-access-for-github-enterprise-importer.
```